index.html

```diff
@@ -1,667 +1,667 @@
-	The object should be modified to reflect the new structure in defined in the update
-	activity.</p>
-    </section>
-    <section id="delete-activity">
-      <h3>Delete Activity</h3>
-      <p>The <code>Delete</code> activity is used to delete an already existing object. This
-	MUST leave a shell of the object that will be displayed in activities which reference
-	the deleted object. If the deleted object is requested the server should respond with
-	the HTTP 410 Gone status code.
-
-	A deleted object:
-<pre class="example">
-  {
-    "@context": "http://www.w3.org/ns/activitystreams",
-    "@id": "https://example.com/~alice/note/72",
-    "@type": "Note",
-    "published": "2015-02-10T15:04:55Z",
-    "updated": "2015-02-10T15:04:55Z",
-    "deleted": "2015-02-10T15:04:55Z",
-  }
-</pre>
-    </section>
-    <section id="follow-activity">
-      <h3>Follow Activity</h3>
-      <p>The <code>Follow</code> activity is used to subscribe to the activities of another user.
-	Once the user has followed a user, activities shared with the Follows of that user
-	SHOULD be added to the actors's inbox.</p>
-    </section>
-    <section id="unfollow-activity">
-      <h3>Unfollow Activity</h3>
-      <p>The <code>Unfollow</code> activity is used to unsubscribe to the activities of actor.
-	Once the user has unfollowed a user, the new activities posted to the Followers collection
-	of that user SHOULD not be added to the actors's inbox.</p>
-    </section>
-    <section id="favorite-like-activity">
-      <h3>Favourite or Like Activity</h3>
-      <p>The <code>Favorite</code> and <code>Like</code> activities are synonyms. These are used to add
-	the actor to list of users who have liked or favorited the object.</p>
-    </section>
-    <section id="unfavorite-unlike-activity">
-      <h3>Unfavorite or Unlike Activity</h3>
-      <p>The <code>Unfavorite</code> and <code>Unlike</code> activity are synonyms. These are used to
-	remove the actor from the list of users who have liked or favorited the object.</p>
-    </section>
-    <section id="add-activity">
-      <h3>Add Activity</h3>
-      <p>The <code>Add</code> activity is used to add the object to the collection specified in the
-	<code>target</code> property of the activity.
-    </section>
-    <section id="remove-activity">
-      <h3>Remove Activity</h3>
-      <p>The <code>Remove</code> activity is used to remove the object from the collection specified in the
-	<code>target</code> property of the activity.
-    </section>
-    <section id="block-activity">
-      <h3>Block Activity</h3>
-      <p>The <code>Block</code> activity is used to indicate that the actor does not want a user (defined in
-	the <code>object</code> property) to be able to interact with objects posted by the user. The
-	server SHOULD prevent the user from interacting with any object posted by the actor.</p>
-    </section>
-  </section>
-  </body>
-</html>
+	<!DOCTYPE html>
```

Comment:
On 2016-02-02 7:54 , Dmitri Zagidulin wrote:
> Is the re-formatting to use Tabs intentional or accidental?

it's accidental, i am sorry about that.